woocommerce/woocommerce-admin
# Create E2E package

See #6807

This PR moves the E2E tests and supporting code to `@woocommerce/admin-e2e-tests` located in `packages/admin-e2e-tests`. There are no functional changes to any of the tests. The changes include

- Exclude `admin-e2e-tests` from the build zip
- Update the `test:e2e` script
  - The test container expects `WP_VERSION` to be an explicit version and defaults to `latest`
  - Add a check for running containers before attempting to stop them
- Addresses all linting issues in the E2E code
- Adds `admin-e2e-tests` to `npm run build`

I set the initial version to `0.1.0` with the intent to release a subsequent version with the remainder of the tasks from #6807.

Note: `jest` prohibits `import`ing its tests functions outside of the jest environment. Any files using jest have Common JS references for those functions.
 
### Detailed test instructions:

-   Review CI run
-   Locally run `npm run test:e2e`

No changelog
